Gaijinizer patch/Map447.txt

```diff
@@ -3,446 +3,570 @@
 > END STRING
 
 > BEGIN STRING
 この結界、あの子が作ったものだと思ってたけど
 消えないところをみると、そうじゃないみたいだし……
 
 この結界って、一体何なんだろう？
-> CONTEXT: Map447/events/1/pages/0/147/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/147/Dialogue/enemy15
+I thought that this barrier over us might have
+been her doing, but seeing as how it's still up
+even after she's gone, I suppose it wasn't. Just
+what on earth is it?#Ron
 > END STRING
 
 > BEGIN STRING
 あなたも……魔力は感じるけど、彼女と同じなのかな？
 
 『実体化』しているっていうより……
 『出てきている』って感じ………
-> CONTEXT: Map447/events/1/pages/0/155/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/155/Dialogue/enemy15
+And you... I can sense magic power within you,
+but it feels similar to hers. It feels less like
+you've materialized, and more like you've
+"appeared".#Ron
 > END STRING
 
 > BEGIN STRING
 さっきは私のことを知っているみたいな話し方してたけど
 
 もしかして、あなたが私をここに呼んだの？
-> CONTEXT: Map447/events/1/pages/0/160/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/160/Dialogue/enemy15
+And you talked about me before like you knew who
+I was. Don't tell me, are you the one who
+summoned me here?#Ron
 > END STRING
 
 > BEGIN STRING
 …………察しがいいね。
 
 ボクや彼女みたいなのは『魔物』……
 さしずめ\\C[2]『モンスター娘』\\C[0]って呼んでくれればいいと思うよ。
-> CONTEXT: Map447/events/1/pages/0/167/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/167/Dialogue/sirousagi_fc1
+Bingo.
 
+That girl and I are "monsters"... for now you can
+just call us \\C[2]monmusumes\\C[0].#Ron
 > END STRING
 
 > BEGIN STRING
 キミたち『悪魔』みたいに、人間界に実体化するのに
 媒体も、生体エネルギーも要らない。
 
 搾精で『繁殖』する個体だって珍しくない。
-> CONTEXT: Map447/events/1/pages/0/172/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/172/Dialogue/sirousagi_fc1
+Unlike you "demons", we don't require energy from
+organisms or a medium to stay materialized in the
+human realm. It's not uncommon for some of us to
+use squeezing humans as a means of reproduction.#Ron
 > END STRING
 
 > BEGIN STRING
 『モンスター娘』……
 
 『魔物』、か……
-> CONTEXT: Map447/events/1/pages/0/177/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/177/Dialogue/kazuya
+Monmusumes...
 
+Monsters...#Ron
 > END STRING
 
 > BEGIN STRING
 あと、この結界はボクが展開したものだよ。
 
 確かに夜深ちゃんの神社を投影してはいるけど 
 この場所は人間界でも、虚数空間でもない…… 
-> CONTEXT: Map447/events/1/pages/0/184/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/184/Dialogue/sirousagi_fc1
+Also, I was the one who made the barrier. But
+while it's true that I'm projecting your shrine,
+this place is neither in the human realm nor in
+imaginary space.#Ron
 > END STRING
 
 > BEGIN STRING
 君は、そこにいる\\N[1]くんを助けてもらうために 
 この結界内に呼ばれた「ゲスト」の一人…… 
 
 ……まぁ、詳しい説明は、全員揃ってからにするつもり
-> CONTEXT: Map447/events/1/pages/0/189/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/189/Dialogue/sirousagi_fc1
+You're one of the "guests" I called here for the
+purpose of saving \\N[1]-kun here. The rest
+of the details can wait until everyone else is
+gathered here, though.#Ron
 > END STRING
```

Comment:
a few too many "heres", let's clean it up a tiny bit for flow:
Additionally, I think help is a better word then save, unless there's a direct implication in the text...
```
You're one of the "guests" I called here for the
purpose of helping \\N[1]-kun. The rest of the
details can wait until everyone else has
finally gathered.
```